src/pgzx/c.zig

```diff
@@ -1,202 +1,7 @@
 const std = @import("std");
 
-const includes = @cImport({
-    @cInclude("postgres.h");
-    @cInclude("postgres_ext.h");
+pub const c = @import("c_translated");
```

Comment:
The introduction of `c_translated` looks good. But I don't think we need the `c.zig` file anymore. By declaring `c` in `c.zig` we still have the funny `pgzx.c.c` module. Instead I was thinking to have `pgzx.c = @import("c_translated")`. This would give us a proper `pgzx.c` module with all symbols in place I think.